app/frontend/src/components/Map/index.css.ts

```diff
@@ -1,6 +1,35 @@
 import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+import { vars } from '@/styles';
 
 export const container = style({
   height: '100%',
   paddingTop: '8.5rem',
 });
+
+export const label = style({
+  padding: '0.2rem',
+  background: vars.color.morakRed,
+  color: vars.color.grayscaleWhite,
+  borderRadius: '0.5rem',
+});
+
+export const marker = recipe({
+  base: {
+    width: '5rem',
+    height: '5rem',
+  },
+  variants: {
+    theme: {
+      green: {
+        fill: vars.color.morakGreen,
+        stroke: vars.color.subGreen,
+      },
+      red: {
+        fill: vars.color.morakRed,
+        stroke: vars.color.grayscale100,
```

Comment:
```suggestion
const { morakGreen, subGreen, morakRed,grayscale100 } = vars.color // 로 미리 선언해 두고 
        fill: morakGreen,
        stroke: subGreen,
      },
      red: {
        fill: morakRed,
        stroke: grayscale100,
```

이렇게 사용하는 것이 좋다고 하네요~!

Reply:
https://github.com/boostcampwm2023/web17_morak/pull/259/commits/6571de052a3a69468fb79a7fd12b1698f50747df
수정하였습니다!